app/app-information/app-information-public/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+  id("hedvig.android.ktlint")
+  id("hedvig.android.library")
+  alias(libs.plugins.squareSortDependencies)
+  alias(libs.plugins.apollo)
+}
+
+dependencies {
+  apolloMetadata(projects.apolloGiraffePublic)
```

Comment:
This is the part that hooks up this module with giraffe, and we get access to the generated apollo classes.